meinberlin/react/contrib/ListCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import django from 'django'
+
+import { ListCardStats } from './ListCardStats'
+
+const translated = {
+  detailsStr: django.gettext('Details'),
+  toStr: django.gettext('to:')
+}
+
+export const ListCard = (props) => {
+  const { item, permissions, children } = props
+
+  return (
+    <li id={'item_' + item.pk}>
+      <article className="modul-card" data-add-clickable-area="smart">
+        <header className="card__header">
+          <h3 className="title">{item.name}</h3>
+          {children}
+        </header>
+        <div className="card__body">
+          <div className="row leftandright center">
+            <div className="left">
+              <ListCardStats
+                permissions={permissions}
+                positiveCount={item.positive_rating_count}
+                negativeCount={item.negative_rating_count}
+                commentCount={item.comment_count}
+                voteCount={item.vote_count}
+              />
+            </div>
+            <div className="center">
+              <a href={item.url} className="more" data-mainlink="true">{translated.detailsStr}
+                <span className="aural">{translated.toStr} {item.name}</span>
+              </a>
+            </div>
+          </div>
+        </div>
+      </article>
+      {/* <div className="list-item__vote">
+        {permissions.has_voting_permission_and_valid_token && item.vote_allowed && (
+          <VoteButton
+            objectID={item.pk}
+            tokenvoteApiUrl={tokenvoteApiUrl}
+            isChecked={item.session_token_voted}
+            onVoteChange={props.onVoteChange}
+            currentPage={props.currentPage}
+            disabled={!props.votesLeft && !item.session_token_voted}
+          />
+        )}
+      </div> */}
```

Comment:
this element has not be confirmed with a design yet so will leave it commented out for now I think